src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -588,6 +602,20 @@ public void InstallPackage(string source, EnvDTE.Project project, string package
                 _packageInstaller.InstallPackage(source, project, packageId, version, ignoreDependencies);
             }
 
+            public void InstallLatestPackage(string source, EnvDTE.Project project, string packageId, bool includePrerelease, bool ignoreDependencies)
+            {
+                // Use reflection until the 3.5.0 version of NuGet.VisualStudio is released
+                // publicly.  Then we call into this method directly.
+                var installLatestPackageInfo = _packageInstaller.GetType().GetMethod(
```

Comment:
Tracking bug for this?

Also - this is always public, just not in the nuget package?  Or we should have some different `BindingFlags`?